src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -125,6 +125,6 @@ protected function _hash(ConnectionInterface $db)
  */
     protected function _getTableKey ()
     {
-        return $this->connection() . '-' . $this->table;
+        return $this->connection() . '-' . $this->table . '-' . static::class;
```

Comment:
We should update the doc block above.

Let's also explain why not only the the table suffixes.
See https://github.com/FriendsOfCake/fixturize/pull/6 why the connection is prepended, for example.

Reply:
Fixed.